commands/Factorio/onlineplayers.js

```diff
@@ -0,0 +1,39 @@
+const { MessageEmbed } = require("discord.js");
+const Command = require("../../base/Command.js");
+const rcon = require("../../helpers/rcon")
+
+class OnlinePlayers extends Command {
+  constructor(client) {
+    super(client, {
+      name: "onlineplayers",
+      description: "Get online players on all servers",
+      usage: "",
+      examples: ["{{p}}onlineplayers"],
```

Comment:
By default, if `examples` is empty, it will add the prefix and command name to it. This line is sort of not needed